test/common/ops/releaseBoth.js

```diff
@@ -51,6 +51,15 @@ describe('shared.ops.releaseBoth', () => {
     expect(user.achievements.triadBingoCount).to.equal(2);
   });
 
+  it('does not grant triad bingo if any pet has not been previously found', () => {
+    let triadBingoCountBeforeRelease = user.achievements.triadBingoCount;
+    user.items.pets[animal] = -1;
+    let [, message] = releaseBoth(user);
```

Comment:
can this be changed to `let message = releaseBoth(user)[1];`? I find it more readable